tests-beta/ios/MASVS-STORAGE/MASTG-TEST-0215.md

```diff
@@ -2,20 +2,21 @@
 platform: ios
 title: Sensitive Data Not Excluded From Backup
```

Comment:
```suggestion
title: Sensitive Data Not Marked For Backup Exclusion
```

Reply:
another test: Sensitive data not written to paths that are backed up by the system